CHARTER.md

```diff
@@ -0,0 +1,117 @@
+
+
+**Technical** **Charter (the “Charter”)**  
+**for**   
+**Besu a Series of LF Projects, LLC**
```

Comment:
Is this just boilerplate or is this charter intended to cover multiple repositories?

Reply:
It covers all of the Besu repositories.